my-react-app/src/app/components/baseComponents/Game/index.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+import Board from '../Board';
+
+import './styles.scss';
+import calculateWinner from './utils.js';
+
+class Game extends React.Component {
```

Comment:
Import `Component` directly from the react package.

`import React,{Component} from 'react';`